sqlalchemy_redshift/dialect.py

```diff
@@ -811,7 +820,9 @@ def _get_all_column_info(self, connection, **kw):
               col_type varchar,
               col_num int)
             ORDER BY "schema", "table_name", "attnum";
-            """)
+            """
+            % generated
+            )
```

Comment:
```suggestion
            """ % generated)
```

This is currently failing tests due to linting on this change.

Reply:
Nice catch